electron/launcher/src/App.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { isSafeToUnpackElectronOnRemoteBuildServer } from 'app-builder-lib/out/platformPackager'; 
-import { Router, Route, Switch, Link} from 'react-router-dom';
+import { Router, Route, Switch } from 'react-router-dom';
 
 import Root from 'components/Root/Root';
 import Login from 'components/Login/Login';
+import Header from './components/Header/Header';
```

Comment:
Please use absolute path when importing, as per @BenjixD order. Root directory for React is the `src` folder.